lms/validation/_lti_launch_params.py

```diff
@@ -174,8 +174,8 @@ def _decode_url(self, _data, **_kwargs):  # pylint:disable=no-self-use
         return _data
 
 
-class ContentItemSelectionLTILaunchSchema(_CommonLTILaunchSchema):
-    """Schema for content item selection LTI launches."""
+class DeeplinkingLTILaunchSchema(_CommonLTILaunchSchema):
```

Comment:
I used the same convetion in the other PR, `Deeplinking`/`deeplinking_` instead of `DeepLinking`/`deep_linking_`. 

I reckon the convetion is to not split "concepts" even if they are two words. We do the same for for example "blackboard_` instead of `black_board_`.

Reply:
I don't think it's anything that fancy. It's just about replacing spaces with underscores or upper case letters. 

"Blackboard" is a product name, so that's why it's all one word. If we were talking about an actual "black board" we would do it the other way, or if their product was called "Bla Ck Boar D" we might do something else.

In the spec https://www.imsglobal.org/specs/lticiv1p0-intro it's always referred to as "deep linking" or "Deep Linking". "deeplinking" doesn't appear in that document and would be something we invented. So I think the correct thing is "deep_linking" or "DeepLinking".